.github/workflows/test-frontend-unit.yml

```diff
@@ -0,0 +1,54 @@
+name: "Frontend test suite"
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - dev
+      - release/*
+    paths:
+      - '**/frontend/**/*.ts'
+      - '**/frontend/**/*.js'
+      - '**/frontend/**/*.json'
+
+  pull_request:
+    types: [opened, reopened, synchronize]
+    paths:
+      - '**/frontend/**/*.ts'
+      - '**/frontend/**/*.js'
+      - '**/frontend/**/*.json'
+
+permissions:
+  contents: read
+
+defaults:
+  run:
+    working-directory: ./frontend
+
+jobs:
+  units:
+    name: Units
+    runs-on: ubuntu-latest
+  
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '18.13'
+          cache: npm
+          cache-dependency-path: frontend/package-lock.json
+
+      - name: Install Dependencies
+        id: npm-i
+        run: npm i
+
+      - name: Register plugins
+        id: npm-run-ci-plugins-register-frontend
+        run: npm run ci:plugins:register_frontend
+
+      - name: Test (Angular)
+        id: npm-test
+        run: npm test -- --code-coverage
```

Comment:
Although test coverage results are written to `stdout` (so visible in the job output), we should probably find a way to append the report to the PR somehow - @machisuji @cbliard any ideas what our options might be?

Reply:
@myabc you can at least append it as a [job summary](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#adding-a-job-summary). Or use one of the existing actions that posts/edits a comment to include any body you would like.

Reply:
Let's merge this as is, we can improve the output when extending test suite to include stimulus (which might result in us using a different test runner overall)